### PR TITLE
Add tables for point boosts

### PR DESCRIPTION
**Pull request by taycaldwell**
Update prisma schema to define model for point boost database tables.

### CODE REVIEW

**Comment by siwan-cb** [comment]
I think the easier way is make changes on supabase and do
`make db-pull` and `make db-generate` on models. I think enums are okay to check-in

@amit-shah-cb thought?

**Comment by amit-shah-cb** [comment]
I see the tables in supabase so thats good; do we want to add points to the boost_configuration so we can change how much they are worth through config?

File: prisma/schema.prisma [hunk 1]
```diff
@@ -98,7 +97,41 @@ model treasure_box_state {
...
+}
+
+/// This model contains row level security and requires additional setup for migrations. Visit https://pris.ly/d/row-level-security for more info.
+model claimed_boost {
+  id                  BigInt              @id @default(autoincrement())
+  updated_at          DateTime            @default(now()) @db.Timestamptz(6)
+  user_address        String              @db.VarChar
+  boost_id            BigInt
+  game_id             BigInt
+  badge_configuration boost_configuration @relation(fields: [boost_id], references: [id], onDelete: Cascade)
```
Comment (amit-shah-cb) [inline prisma/schema.prisma:121]: boosts and badges are related?

Comment (amit-shah-cb) [reply prisma/schema.prisma:121]: OH i think you want to rename this property to `boost_configuration`

File: prisma/schema.prisma [hunk 2]
```diff
@@ -98,7 +97,41 @@ model treasure_box_state {
...
+  id                 BigInt          @id @default(autoincrement())
+  name               String          @db.VarChar
+  contract_addresses String[]        @db.VarChar
+  refresh_time       DateTime?       @db.Time(6)
+  image_url          String?         @db.VarChar
+  game_id            BigInt          @default(0)
+  cta_url            String?         @db.VarChar
+  cta_text           String?         @db.VarChar
+  boost_type         boost_type      @default(NFT)
+  claimed_boost      claimed_boost[]
```
Comment (amit-shah-cb) [inline prisma/schema.prisma:111]: you probably want an is_valid field here so you can turn it on and off

File: prisma/schema.prisma (hunk 1 above)
Comment (taycaldwell) [reply prisma/schema.prisma:121]: Yeah, I initially misclicked badge_configuration in the Supabase UI, and it looks like re-pulling updates everything but the name lol.

**Pull request closed without merging**